### PR TITLE
Add events for dbt feature

### PR DESCRIPTION
**Pull request by dlawin**
Adds new events for the `--cloud` option

Sets a new entrypoint `CLI-dbt` for `--dbt` and `--cloud`

Added additional parameters to end_event in order to differentiate `--cloud` events

Most of the information is hardcoded to 0/None/etc. on the end event as we don't currently have the related information. Since I think it's likely we will be adding polling and diff results for `--cloud` in the near future, I thought it made sense to reuse the existing event structure.

### CODE REVIEW

File: data_diff/utils.py [hunk 1]
```diff
@@ -72,3 +72,8 @@ def get_timestamp(_match):
         return datetime.now().isoformat("_", "seconds").replace(":", "_")
 
     return re.sub("%t", get_timestamp, name)
+
+
+def truncate_error(error: str):
```
Comment (dlawin) [inline data_diff/utils.py:77]: moved this from diff_tables.py so I can reuse it

File: data_diff/tracking.py [hunk 2]
```diff
@@ -85,12 +85,14 @@ def create_start_event_json(diff_options: Dict[str, Any]):
 def create_end_event_json(
     is_success: bool,
     runtime_seconds: float,
-    db1: str,
-    db2: str,
+    data_source_1_type: str,
```
Comment (dlawin) [inline data_diff/tracking.py:88]: I changed these just because I found the names confusing/misleading

Comment (kylemcnair) [reply data_diff/tracking.py:88]: non-blocking: I'll need to update downstream reporting

Comment (dlawin) [reply data_diff/tracking.py:88]: this just impacts the python code, the actual event is not affected

Comment (kylemcnair) [reply data_diff/tracking.py:88]: oh I see, still "data_source_1_type" 👍

**Review by kylemcnair — APPROVED** [review]
Screen Shot 2023-02-07 at 3 55 41 PM
Events with the new properties look good in Snowflake

Adding approval from that perspective, deferring to others tagged for review of the code

**Comment by erezsh** [comment]
@dlawin Looks okay to me. Did you run black?

**Comment by williebsweet** [comment]
> @dlawin Looks okay to me. Did you run black?

@erezsh Dan is on PTO today. Can you pull it down and do that?

**Comment by erezsh** [comment]
Done

**Pull request merged**